app/views/shared/summary/_textual_multilabel.html.haml

```diff
@@ -12,4 +12,11 @@
       %tr
         - item.each do |value|
           %td
-            = value
+            - if value && items[:expandable] && value.length > 50
```

Comment:
@alongoldboim I hate hard-coded values... any idea on how to drop this empirical value (50) in favor of something more automated? Is it something that the js side can decide depending on the cell size? Maybe you can find something interesting here: http://www.google.com/search?q=jquery%20cell%20overflow